sorts.py

```diff
@@ -68,10 +68,33 @@ def chris_insert_sort(values):
 
 
 def peter_insert_sort(values):
-    # TODO: Test for empty input.
-    # TODO: Build sorted linked list from input using Link class.
-    # TODO: Copy sorted linked list into Python list.
-    pass
+    newlist = []
+    if len(values) <= 1:
+        return values
+    else:
```

Comment:
Rather than nest the rest of the function inside of an else: block, since the if: block returns you don't need an else: or extra indentation at all.